### PR TITLE
Dependency Sweep

### PR DESCRIPTION
**Pull request by darthtrevino**

### CODE REVIEW

File: packages/scripts/src/entry.mts [hunk 1]
```diff
@@ -66,21 +65,23 @@ async function bootstrap(command: string) {
...
 	}
 
 	// error on unknown commands
 	program.on('command:*', () => {
-		console.log(
-			'Invalid command: %s\nSee --help for a list of available commands.',
-			program.args.join(' '),
-		)
-		process.exit(1)
+		// console.log(
```
Comment (natoverse) [inline packages/scripts/src/entry.mts:80]: Did you mean to leave this commented out?

File: packages/scripts/src/steps/webpack/getConfig.mts [hunk 2]
```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft. All rights reserved.
  * Licensed under the MIT license. See LICENSE file in the project.
  */
-import { configure } from '@essex/webpack-config'
+import essexWebpackConfig from '@essex/webpack-config'
```
Comment (natoverse) [inline packages/scripts/src/steps/webpack/getConfig.mts:5]: Moving to default exports?

File: packages/scripts/src/entry.mts (hunk 1 above)
Comment (darthtrevino) [reply packages/scripts/src/entry.mts:80]: I meant to delete it; pushing an update

File: packages/scripts/src/steps/webpack/getConfig.mts (hunk 2 above)
Comment (darthtrevino) [reply packages/scripts/src/steps/webpack/getConfig.mts:5]: I noticed some issues with importing CJS from ESM - I think it's from TypeScript. 

But basically, named imports from CJS were causing issues and we had to destructure later.

**Pull request merged**